src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -1966,6 +2038,47 @@ private void CopyEditedPropertiesToSpecies(MicrobeSpecies target)
         }
     }
 
+    private void SetLightLevelOption(string option)
+    {
+        var selection = (LightLevelOption)Enum.Parse(typeof(LightLevelOption), option);
+
+        GUICommon.Instance.PlayButtonPressSound();
+
+        selectedLightLevelOption = selection;
+        ApplyLightLevelOption();
+    }
+
+    private void ApplyLightLevelOption()
+    {
+        // Show selected
```

Comment:
Shouldn't this unset the pressed values first and then set one to true? Similarly to all of the other tab-style button management already in the game.